src/Plugin/Field/FieldWidget/OgComplex.php

```diff
@@ -63,7 +52,9 @@ public function elementValidate($element, FormStateInterface $form_state, $form)
       return;
     }
 
-    $entity = entity_load($this->getFieldSetting('target_type'), $matches[1]);
+    $entity = \Drupal::entityManager()
```

Comment:
is this `elementValidate` really needed - isn't the selection handler validation taking care of this (I couldn't find the D7 origin of this)?